Makefile

```diff
@@ -57,12 +57,16 @@ verify-shellcheck: ## Runs shellcheck
 ##@ Tests
 
 .PHONY: test
-test: test-go test-sh ## Runs unit tests to ensure correct execution
+test: test-go-unit test-sh ## Runs unit tests to ensure correct execution
 
-.PHONY: test-go
-test-go: ## Runs all golang tests
+.PHONY: test-go-unit
+test-go-unit: ## Runs golang unit tests
 	./hack/test-go.sh
 
+.PHONY: test-go-integration
+test-go-integration: ## Runs golang integration tests
+	GO111MODULE=on go test -v -timeout="15m" -count=1 ./cmd/...
```

Comment:
Let's put this into a script in the same way like `test-go.sh`. Or we pass a flag to `test-go.sh` to indicate that we either run the unit or the integration tests.